### ISSUE TITLE
Group by in Histogram

### ISSUE THREAD

**Issue by harshinim** [lucidworks/banana#161]
Can we group by more than one field in histogram? I know that this is not currently supported, but is it possible to add this feature?

**Comment by aadel** [comment]
Hi, this seems interesting feature! It's possible indeed to group by multiple fields and add support for it.

**Comment by nickvasilyev** [comment]
I would really like this too, but I am not sure how to do this in Solr. Is
there a way to do a date range facet pivot?

**Comment by aadel** [comment]
@wrdrvr actually, Solr supports multiple group fields in a single request.

**Comment by nickvasilyev** [comment]
Right, I apologize, I responded in a hurry. I was thinking of using the histogram to generate a line chart for counts based on  facet pivot instead of what it currently does. Solr supports this, but I don't think you can get Solr to split it up by date ranges. I've had to round down by dates to a day and hour in separate fields to get this to work.
